featuretools/entityset/entityset.py

```diff
@@ -1113,6 +1117,19 @@ def plot(self, to_file=None):
                               ' (See https://docs.featuretools.com/getting_started/install.html for' +
                               ' details)')
 
+        # Try rendering a dummy graph to see if a working backend is installed
+        try:
+            graphviz.Digraph().pipe()
+        except graphviz.backend.ExecutableNotFound:
+            raise RuntimeError(
+                "To plot entity sets, a graphviz backend is required.\n" +
+                "Install the backend using one of the following commands:\n" +
+                "  Mac OS: brew install graphviz\n" +
+                "  Linux (Ubuntu): sudo apt-get install graphviz\n" +
+                "  Windows: conda install python-graphviz"
+            )
+            return  # Exit plotting method at this point
```

Comment:
no need for the return after raise the RuntimeError

Reply:
Fair enough. Removed the *return* statement with 6df0764.